lib/results_screen.dart

```diff
@@ -0,0 +1,139 @@
+import 'package:distal_radius/image_handler.dart';
+import 'package:flutter/material.dart';
+
+import "screen_button.dart";
+
+import 'dart:io';
+
+class ResultsScreen extends StatefulWidget {
+  const ResultsScreen({super.key});
+
+  static const String id = "results_screen";
+
+  @override
+  State<ResultsScreen> createState() => _ResultsScreenState();
+}
+
+// State class that handles logic in the screen
+// contains all functions relevant to business logic
+class _ResultsScreenState extends State<ResultsScreen> {
+  ImageHandler imageHandler = ImageHandler();
+
+  void cancelResults() {
+    print("Cancelled");
+  }
+
+  void toExport() {
+    print("Send to export");
+  }
+
+  Widget getHeader() {
+    return Stack(
+      alignment: Alignment.center,
+      children: const [
+        Positioned(left: 10, child: BackButton()),
```

Comment:
Again, the same issue as the initial UI screen. We probably don't want this back button, which can mess up the analysis. The home button at the bottom of the screen should be enough.